scripts/generate_api_client_silicon.sh

```diff
@@ -14,7 +14,8 @@ docker run \
     --network="host" \
     --rm \
     --volume "${PWD}:/local" \
-    "parsertongue/swagger-codegen-cli:${SWAGGER_CODEGEN_CLI_VERSION:-'latest'}" generate \
+    -v $JAVA_HOME/lib/security/cacerts:/opt/java/openjdk/lib/security/cacerts \
```

Comment:
je recommande de toujours utiliser les options longues quand c'est possible car c'est plus explicite et évite les ambiguités

je recommande aussi de mettre les variables entre `"` au cas où il y aurait des espaces pour éviter que ça soit traité comme plusieurs arguments

```suggestion
    --volume "$JAVA_HOME/lib/security/cacerts:/opt/java/openjdk/lib/security/cacerts" \
```